Kickstarter-iOS/Views/Controllers/SettingsAccountViewController.swift

```diff
@@ -199,13 +200,31 @@ extension SettingsAccountViewController: UITableViewDelegate {
   }
 
   func tableView(_ tableView: UITableView, heightForFooterInSection section: Int) -> CGFloat {
-    return 0.1
+
```

Comment:
Has there been a consensus if we prefer to open functions, etc. with empty line or is this mostly a personal preference?

Reply:
It's used to add a line between the function declaration and its implementation to make the reading easier, but I've been avoiding this on our codebase. So I would say that these lines are there by accident :)